src/quicktype-core/language/CPlusPlus.ts

```diff
@@ -923,98 +1051,183 @@ export class CPlusPlusRenderer extends ConvenienceRenderer {
         });
     }
 
+    protected emitClassHeaders(className: Name): void {
+        const ourQualifier = this.ourQualifier(true) as string;
+        this.emitLine("void from_json(const json& _j, ", ourQualifier, className, "& _x);");
+        this.emitLine("void to_json(json& _j, const ", ourQualifier, className, "& _x);");
+    }
+
     protected emitClassFunctions(c: ClassType, className: Name): void {
-        const ourQualifier = this.ourQualifier(true);
+        const ourQualifier = this.ourQualifier(true) as string;
+
+        let EncodeKeyStart: string = "";
```

Comment:
Local variables always start with lowercase. `EncodeKeyStart`.  There are many other cases that need to be fixed, too.

In reference to above, this should be `Sourcelike`, not `string`.